portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,30 @@
 <!DOCTYPE html>
-<html>
+<html style="background-color:Tomato;">
   <head>
+    <style>
+      .myDiv {
+  border: 5px outset red;
+  background-color: lightblue;    
+  text-align: center; }
+  </style>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>  Taniece's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <div class="myDiv">
```

Comment:
Style guide wants class and id names to be dashed. 
See Google HTML style guide

Reply:
Looks like this still needs to be updated

Reply:
And mydiv isnt very descriptive for a class name